app/lib/menu/menu-builder.js

```diff
@@ -221,6 +221,7 @@ class MenuBuilder {
 
     this.menu.append(new MenuItem({
       label: 'Export As Image',
+      accelerator: 'CommandOrControl+SHIFT+E',
```

Comment:
The string format differs from other menu items, let's be consistent here

```js
accelerator: 'CommandOrControl+Shift+E'
```

Reply:
@pinussilvestrus I have made the changes.